### PR TITLE
[WFLY-19223] LayersTestCase should pass with new org.wildfly.security.elytron-dynamic-ssl module

### PR DESCRIPTION
**Pull request by skyllarr**
https://issues.redhat.com/browse/WFLY-19223

Requires: https://github.com/wildfly/wildfly-core/pull/5947
____

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

____

### CODE REVIEW

**Comment by Skyllarr** [comment]
@yersan this PR solves the LayersTestCase failures happening on https://github.com/wildfly/wildfly-core/pull/5947

File: testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java [hunk 1]
```diff
@@ -54,6 +54,7 @@ public abstract class LayersTestBase {
             "org.jboss.as.xts",
             // TODO should an undertow layer specify this?
             "org.wildfly.event.logger",
+            "org.wildfly.security.elytron-dynamic-ssl",
```
Comment (yersan) [inline testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java:57]: In my opinion, this is not the expected fix.

`test-all-layers` installation contains the `elytron` galleon layer, so the `org.wildfly.security.elytron-dynamic-ssl` should have been provisioned by the `test-all-layers`, which makes this change unnecessary.

The fix should be in WildFly Core PR, I'll comment more about it there. The problem is that `elytron` galleon layer is not provisioning `org.wildfly.security.elytron-dynamic-ssl` package.

**Review by yersan — CHANGES_REQUESTED** [review]
@Skyllarr I think this PR can be closed, in my opinion, this is not the expected fix.

File: testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java (hunk 1 above)
Comment (yersan) [reply testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java:57]: @bstansberry I guess we also are going to need a solution to tweak all of these galleon layers tests based on provisioning stability levels. Not sure if we have already discussed the approach for them.

**Comment by Skyllarr** [comment]
@yersan Thank you for looking into this and explaining, I'm closing this PR

**Pull request closed without merging**